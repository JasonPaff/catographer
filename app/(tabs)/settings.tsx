import { Settings } from "../../components/settings-screen/settings";

export default function SettingsScreen() {
	return <Settings />;
}
