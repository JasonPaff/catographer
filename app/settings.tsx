import { Text, View } from "tamagui";

export default function SettingsScreen() {
	return (
		<View alignItems={"center"} flex={1} justifyContent={"center"}>
			{process.env.EXPO_PUBLIC_DEV === "true" && <Text fontSize={20}>Settings Screen</Text>}
		</View>
	);
}
