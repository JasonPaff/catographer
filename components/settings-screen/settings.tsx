import { StyleSheet, Text, View } from "react-native";

export function Settings() {
	return (
		<View style={styles.container}>
			<Text style={styles.heading}>Settings Screen</Text>
		</View>
	);
}

const styles = StyleSheet.create({
	container: {
		alignItems: "center",
		flex: 1,
		justifyContent: "center",
	},
	heading: {
		color: "white",
		fontSize: 20,
	},
});
