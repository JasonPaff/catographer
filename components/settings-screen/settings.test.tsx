import { render, screen } from "@testing-library/react-native";
import { Settings } from "./settings";

test("<Settings/> should render correctly", () => {
	render(<Settings />).toJSON();
	expect(screen).toMatchSnapshot();
});
