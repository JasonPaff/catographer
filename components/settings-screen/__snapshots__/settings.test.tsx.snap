// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Settings/> should render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "color": "white",
        "fontSize": 20,
      }
    }
  >
    Settings Screen
  </Text>
</View>
`;
